ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-conformance.yaml

```diff
@@ -17,9 +17,11 @@ parameters:
 - name: CLUSTER_TYPE
   value: "azure"
   required: true
+- name: TEST_SUITE
```

Comment:
Why are you addin the `TEST_SUITE*` back?  They were intentionally removed in #1261.

Reply:
I was not aware of this.

Reply:
done